lib/tutorials/en_US/routing.md

```diff
@@ -1,61 +1,73 @@
-## Routing
+# Routing
 
-_This tutorial is compatible with hapi v17_
 
-When defining a route in hapi, as in other frameworks, you need three basic elements: the path, the method, and a handler. These are passed to your server as an object, and can be as simple as the following:
+- [Overview](#overview)
+- [Methods](#methods)
+- [Path](#path)
+- [Optional Parameters](#optionalParameters)
+- [Multi-Segment Parameters](#multiParameters)
+- [Query Parameters](#query)
+- [Request Payload](#requestpayload)
+- [Handler](#handler)
+- [Options](#options)
+- [404 Handling](#missing)
+        
 
-```javascript
+## <a name="overview" /> Overview
+
+When defining a route in hapi, you need three basic elements: the path, the method, and a handler. These are passed to your server as an object, and can be as simple as the following:
+
+```js
 server.route({
     method: 'GET',
     path: '/',
-    handler: function (request, h) {
+    handler: (request, h) => {
```

Comment:
Also why switching from function to arrow functions? I'm concerned that for newcomers they would use arrow functions everywhere and then loose the ability to access properties set by `server.bind`.

Reply:
Makes sense. Fixing now, will resubmit in a few